ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -996,6 +996,13 @@ keys:
         transceiver:
           type: dict
           keys:
+            frequency:
+              type: int
+              convert_types:
+                - str
+              min: 190000
+              max: 200000
```

The value is actually a float in EOS, but we do not support floats natively, so we normally use str and auto-convert from float if given.
```suggestion
              type: str
              convert_types:
                - int
                - float
```